docs/decisions/0005-optional-import-of-confluent-kafka.rst

```diff
@@ -0,0 +1,43 @@
+4. Optional import of confluent-kafka
+====================================
+
+Status
+------
```

Comment:
Nit/Question: Should we use the new RST heading standards? It would be less consistent with earlier docs, but would cause less fixes required down the road (unless that process gets better automated). See https://docs.openedx.org/en/latest/documentors/references/quick_reference_rst.html